test/ANNOTATIONS.yaml

```diff
@@ -334,6 +334,10 @@ jacobi:
     - add dead string literal elimination (#3120)
   04/01/16:
     - casting bug fix (#3615)
+  08/19/16:
+    - Make denormalize on-by-default (#4360)
+  08/20/16:
+    - Revert #4360 (#4367)
```

Comment:
"revert" => "Turn denormilization off-by-default" since the PR number won't mean a whole lot just looking at the graph annotation?